docs/index.md

```diff
@@ -3,10 +3,10 @@
 Ballerina is the premier programming language designed for integration. It allows you to connect apps and services to handle all types of integration scenarios, such as collecting top tweets from a specific location and adding them to a Google spreadsheet, generating emails with real-time quotes pulled from a stock quote service, transforming and routing data based on advanced logic, and much more. 
 
 ## Flexible
-You can build your integrations from sequence diagrams, or write code in the Ballerina language or Swagger. You can also use plug-ins to write Ballerina code in IntelliJ IDEA, Vim, and more. Ballerina is open source, so you can see exactly how it works and join the growing community of users contributing to its code base. And its API allows you to leverage its functionality from other programs.
+You can build your integrations from sequence diagrams, or write code in the Ballerina language or in Open API Specification. You can also use plug-ins to write Ballerina code in IntelliJ IDEA, Vim, and more. Ballerina is open source, so you can see exactly how it works and join the growing community of users contributing to its code base. And its API allows you to leverage its functionality from other programs.
```

Comment:
OpenAPI is a single name ... Lets remove the space. 

What if we say "using OpenAPI Specification"

Reply:
Resloved with 81d9190